ghost/core/core/server/services/bulk-email/bulk-email-processor.js

```diff
@@ -80,14 +79,14 @@ module.exports = {
             .getFilteredCollectionQuery({filter: `email_id:${emailId}+status:[pending,failed]`}, knexOptions)
             .select('id', 'member_segment');
 
-        const batchResults = await Promise.map(batchIds, async ({id: emailBatchId, member_segment: memberSegment}) => {
+        const batchResults = await Promise.all(batchIds.map(async ({id: emailBatchId, member_segment: memberSegment}) => {
             try {
                 await this.processEmailBatch({emailBatchId, options, memberSegment});
                 return new SuccessfulBatch(emailBatchId);
             } catch (error) {
                 return new FailedBatch(emailBatchId, error);
             }
-        }, {concurrency: 10});
+        }, {concurrency: 10}));
```

Comment:
This would not be valid because native `Promise.all` does not take `{concurrency: X}` as a second argument

Reply:
Also, is this code legacy now @SimonBackx?

Reply:
This is indeed legacy and not used as long as emailStability is a GA flag